development.cfg

```diff
@@ -12,6 +12,9 @@ parts +=
 eggs +=
     plone.app.dexterity
     plone.formwidget.autocomplete
+    ftw.testbrowser [tests]
+
+zcml += ftw.testbrowser.tests.views
```

Comment:
I'd remove the changes in `development.cfg` as this was only used for debugging :wink: